ofl/oswald/METADATA.pb

```diff
@@ -2,7 +2,7 @@ name: "Oswald"
 designer: "Vernon Adams, Kalapi Gajjar, Cyreal"
 license: "OFL"
 category: "SANS_SERIF"
-date_added: "2012-02-29"
+date_added: "2021-08-16"
```

Comment:
This should stay the same. Are you using `gftools add-font`?

Reply:
Yeah, I have been. Not sure why it changed that field. I'll correct it.